analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ConditionalStructureSameImplementation_If.cs

```diff
@@ -78,5 +78,25 @@ public void Test_MultilineBlocks()
                 DoSomething1();
             }
         }
+
+        public void Test_Overloads()
+        {
+            int foo = 0;
+            if (someCondition1)
+            {
+                foo++;
+                foo = foo.FooInt(); // FN
```

Comment:
Nice finding!

Reply:
The case of the extension methods?
It was Martin not me 😄!